src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_FplnRecallPage.js

```diff
@@ -3,6 +3,14 @@ class CJ4_FMC_FplnRecallPage {
         let url = "https://www.simbrief.com/api/xml.fetcher.php?userid=" + pilotId + "&json=1";
         let json = "";
 
+        let parseAirport = (icao) => {
+            if (icao.startsWith("K") && (/\d/.test(icao))) {

```

Comment:
```suggestion
            if (/^K\d/.test(icao)) {
```

Reply:
I'm also confused on that. So if it's `KH2L`, we want to strip out the K also?

Reply:
> I'm also confused on that. So if it's `KH2L`, we want to strip out the K also?

yes. the game wants "H2L "  it doesnt accept KH2L

Reply:
> Actually, I'm not sure what you meant by "If there is a number after K". If it starts with K and there's a number anywhere, or literally if there's a number "after K"?

Thanks for the suggestion of doing it smarter. I will gladly take it ;)

Reply:
So. The number can be anywhere in the ident after the K. So the test as suggested wouldn't work.
Needs optional other characters in between

Reply:
How about 
![image](https://user-images.githubusercontent.com/2995606/102907266-1b502900-4476-11eb-8e2d-f5cdb7ac5004.png)

Reply:
Was just going to suggest that, yep. Looks good.